src/sonic-py-common/sonic_py_common/general.py

```diff
@@ -23,3 +24,33 @@ def load_module_from_source(module_name, file_path):
     sys.modules[module_name] = module
 
     return module
+
+
+def getstatusoutput_noshell(cmd):
```

Comment:
>getstatusoutput_noshell [](http://example.com/codeflow?start=4&length=23)

Please add unit tests to cover new functions.

Reply:
Sure, working on UT